lib/context.js

```diff
@@ -517,9 +526,19 @@ api.createTermDefinition = ({
     }
 
     if(reverse.match(KEYWORD_PATTERN)) {
```

Comment:
What's an example where this would match?  Looks like the above checks wouldn't allow `{"@reverse": "@foo"}` or similar to get this far.

Reply:
Previous checks don't specifically look at `@reverse`, except to verify that it's a string.

Reply:
It checks it's a string, `_expandIris` it, then `_isAbsoluteIri` checks that.  Seems like a keyword like string wouldn't get beyond those steps.  Is there an example where it would?

Reply:
Just created https://github.com/w3c/json-ld-api/pull/389, as the order of checks was inverted.